### ISSUE TITLE
How to switch between sessions

### ISSUE THREAD

**Issue by HasinduLanka** [CypherpunkArmory/UserLAnd#1208]
# Describe the issue you are having.

When we open multiple sessions from sessions tab, we can see them running simultaneously. We can close last one with exit command and switch back. 
But how to switch between sessions without closing them?

# Describe how your task behaves in a regular Linux environment

We can press ctrl+shift+F1-6 to run multiple sessions

**Comment by bones4cats** [comment]
try swiping left from the ssh and you will find the sessions

**Comment by l3laze** [comment]
Sessions are accessible in a drawer from the left, but only while in a session. Just drag from the left edge of the screen to see the current sessions and tap one to switch to it. Can also create a new session or show/hide the keyboard.

**Comment by rudolphpienaar** [comment]
The problem with that workflow is that some devices have a "drag from left" mapped to some launcher action. I am not able to access other sessions using this gesture without disabling I suppose that gesture in my launcher. It would be super useful to have this also mapped some keyboard shortcut -- most of my userland interaction is via a bluetooth keyboard so I do think a keyboard mapping is useful.

**Comment by Sir-Erwin** [comment]
I agree with @rudolphpienaar . A keyboard shortcut would be awesome.

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
